src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It seems like there is a syntax error due to a missing closing quote for the string. Make sure to close the string with the appropriate quote to fix the syntax error.